data-avro/src/test/java/com/linkedin/data/avro/TestSchemaTranslator.java

```diff
@@ -3323,6 +3324,19 @@ public void testAvroUnionModeChaining() throws IOException
 enterpriseAccount making the request. Since this parameter is an array, you will need to specify a zero-based index per value.\",\"default\":null,\"compliance\":{\"AggregationCriteria\":\"INHERITED\"}},{\"name\":\"sortBy\",\"type\":[\"null\",{\"type\":\"record\",\"name\":\"MetricsSortBy\",\"doc\":\"The sorting criteria of the report\",\"fields\":[{\"name\":\"engagementMetricType\",\"type\":[\"null\",{\"type\":\"enum\",\"name\":\"EngagementMetricType\",\"doc\":\"Specifies the type of engagement metric for the enterprise entity\",\"symbols\":[\"SECONDS_VIEWED\",\"COMPLETIONS\",\"LOGINS\",\"MARKED_AS_DONE\",\"PROGRESS_PERCENTAGE\",\"VIEWS\",\"ENGAGED_LEARNERS\",\"CERTIFICATE_DOWNLOAD\",\"CERTIFICATE_UNITS_EARNED\",\"ENGAGED_AT\",\"DAYS_ACTIVE\"],\"symbolDocs\":{\"SECONDS_VIEWED\":\"Number of seconds a content has been viewed by enterprise entities\",\"COMPLETIONS\":\"Number of times the content has been completed by enterprise entities\",\"LOGINS\":\"Number of logins for the enterprise entity\",\"MARKED_AS_DONE\":\"Number of times the content has been marked as done by unique enterprise entities\",\"PROGRESS_PERCENTAGE\":\"Percentage of the content completed\",\"VIEWS\":\"Number of views for the content\",\"ENGAGED_LEARNERS\":\"Number of times the content has been engaged with by enterprise entities\",\"CERTIFICATE_DOWNLOAD\":\"Metric type indicating certificate got downloaded by the learner.\",\"CERTIFICATE_UNITS_EARNED\":\"Metric type indicating how many CEU units got earned by the learner.\",\"ENGAGED_AT\":\"Time of engagement in minutesSinceEpoch\",\"DAYS_ACTIVE\":\"Total number of days the user did at least one engagement for any content.\"}}],\"doc\":\"The engagement metric type of the requested report\",\"default\":null,\"compliance\":\"NONE\"},{\"name\":\"engagementMetricQualifier\",\"type\":{\"type\":\"enum\",\"name\":\"EngagementMetricQualifier\",\"doc\":\"Qualifies the engagement metric type\",\"symbols\":[\"TOTAL\",\"UNIQUE\"],\"symbolDocs\":{\"TOTAL\":\"Number of times the content has been engaged by enterprise entities\",\"UNIQUE\":\"Number of times the content has been engaged by unique enterprise entities\"}},\"doc\":\"The engagement metric qualifier of the requested report\",\"default\":\"TOTAL\",\"compliance\":\"NONE\"}]}],\"default\":null,\"compliance\":{\"MetricsSortBy\":\"INHERITED\"}},{\"name\":\"sortOrder\",\"type\":[\"null\",{\"type\":\"enum\",\"name\":\"SortOrder\",\"symbols\":[\"ASCENDING\",\"DESCENDING\"],\"symbolDocs\":{\"ASCENDING\":\"Sort by ascending order\",\"DESCENDING\":\"Sort by descending order\"}}],\"doc\":\"The sort order of the requested report\",\"default\":null,\"compliance\":\"NONE\"},{\"name\":\"locale\",\"type\":[\"null\",\"string\"],\"doc\":\"user locale used to generate the report name. When it is missing the report uses default locale en_US\",\"default\":null,\"compliance\":\"NONE\"}],\"schemaType\":\"DocumentSchema\",\"version\":10,\"upconvertVersion\":10,\"evolutionSafetyMode\":\"IGNORE_WARNINGS\"}";
```

Comment:
Thanks for adding the unit test! 

Just thinking this might be too specific and too detailed, can we add a small represetntive unit test? (and maybe does not show too much linkedin-related info)

Reply:
Also if we can format the test like other tests (adding proper indentations), that will be awesome

Reply:
Done.